### PR TITLE
[ISSUE #3740] add upload file example 

### PR DESCRIPTION
**Pull request by 847850277**
Dear Community:
I have finished [#3740](https://github.com/apache/shenyu/issues/3740) task 1、3、4、6
please review code.

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by dragon-zhang** [comment]
maybe better put `UploadController` to `org.apache.shenyu.examples.common.UploadController` in `shenyu-examples-common` module.

**Comment by 847850277** [comment]
ok、I try to adjust it.

**Comment by yu199195** [comment]
Hi， maybe you can add integration test？

**Comment by dragon-zhang** [comment]
> ok、I try to adjust it. when `UploadController ` to `shenyu-examples-common` module. `shenyu-examples-common` need to import maven dependency `spring-boot-starter-webflux` and `spring-boot-starter-web` and `shenyu-spring-boot-starter-client-springmvc` ? i try in local. it dosn`t work. i think UploadController can not as a common controller. webflux container mabye is nettty, mvc container mabye is tomcat. there are some differences between them.

There are some solutions like `bytebuddy`, but they are too complex. Sorry, forget it.

**Comment by yu199195** [comment]
> ok、I try to adjust it. when `UploadController ` to `shenyu-examples-common` module. `shenyu-examples-common` need to import maven dependency `spring-boot-starter-webflux` and `spring-boot-starter-web` and `shenyu-spring-boot-starter-client-springmvc` ? i try in local. it dosn`t work. i think UploadController can not as a common controller. webflux container mabye is nettty, mvc container mabye is tomcat. there are some differences between them.

>>There are some solutions like `bytebuddy`, but they are too complex. Sorry, forget it.

I think , add integration test is good for this.

**Comment by 847850277** [comment]
i will add this .

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            BufferedWriter bufferedWriterOne = Files.newBufferedWriter(pathOne);
+            bufferedWriterOne.write("111");
+            bufferedWriterOne.flush();
+            bufferedWriterOne.close();
+            BufferedWriter bufferedWriterTwo = Files.newBufferedWriter(pathTwo);
+            bufferedWriterTwo.write("222");
+            bufferedWriterTwo.flush();
+            bufferedWriterTwo.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (dragon-zhang) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java:62]: use `Logger`  to replace `e.printStackTrace()`

Comment (dragon-zhang) [reply shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java:62]: good idea to create and write new file !

**Comment by dragon-zhang** [comment]
> > ok、I try to adjust it. when `UploadController ` to `shenyu-examples-common` module. `shenyu-examples-common` need to import maven dependency `spring-boot-starter-webflux` and `spring-boot-starter-web` and `shenyu-spring-boot-starter-client-springmvc` ? i try in local. it dosn`t work. i think UploadController can not as a common controller. webflux container mabye is nettty, mvc container mabye is tomcat. there are some differences between them.
> 
> > > There are some solutions like `bytebuddy`, but they are too complex. Sorry, forget it.
> 
> I think , add integration test is good for this.

sure, as for complex part, I will try.

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java (hunk 1 above)
Comment (847850277) [reply shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java:62]: i will fix this

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UploadControllerTest {
+
+    private static final String FILE_PATH_NOE = "1.txt";
```
Comment (damonxue) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/UploadControllerTest.java:40]: maybe u want write FILE_PATH_ONE.

**Review by dragon-zhang — APPROVED** [review]

**Comment by dragon-zhang** [comment]
@847850277 nice work !

**Pull request merged**